src/terraform-provider-signalform/signalform/heatmap_chart.go

```diff
@@ -132,7 +132,7 @@ func heatmapChartResource() *schema.Resource {
 						"thresholds": &schema.Schema{
 							Type:        schema.TypeList,
 							Required:    true,
-							Elem:        &schema.Schema{Type: schema.TypeInt},
+							Elem:        &schema.Schema{Type: schema.TypeFloat},
```

Comment:
I think you need to do 3 more things:
1. change thresholds type for single_value_chart
2. change thresholds list type in util.go (line 82)
3. change threshold cast type in util.go (line 84)